.ci/Jenkinsfile

```diff
@@ -436,8 +436,8 @@ def getSmartTAVContext() {
         withGithubNotify(context: 'Linting') {
           deleteDir()
           unstash 'source'
-          script {
-            docker.image('node:12').inside("-v ${WORKSPACE}/${BASE_DIR}:/app"){
+          docker.image('node:12').inside("-v ${WORKSPACE}/${BASE_DIR}:/app"){
```

Comment:
~docker step needs the script step,~ you can remove the inside parameters and change `withEnv("HOME=/app")` by `withEnv("HOME=${WORKSPACE}")` the workspace is mounted as volume and set as working directory